### ISSUE TITLE
intro: premature introduction of workflows?

### ISSUE THREAD

**Issue by simonpcouch** [tidymodels/workshops#418]
This might be more of a request for advice than a proposed change, but at the point that [this slide](https://github.com/tidymodels/workshops/blob/8e67062d6c53ae38ffdcfe804c90bb4c80012b82/slides/intro-03-what-makes-a-model.qmd#L435) is introduced, I don't know that participants would have the vocabulary to understand the explanations here:

<img width="1921" height="732" alt="Image" src="https://github.com/user-attachments/assets/9b0b289b-efb6-421b-8672-5a2769b8f795" />

With an eye to [the annotations](https://github.com/tidymodels/workshops/blob/8e67062d6c53ae38ffdcfe804c90bb4c80012b82/slides/annotations.qmd#L94) for the slide:

* We haven't introduced metrics yet.
* We haven't introduce resampling yet.
* Presumably, the model is just being fitted on the whole training set at this point anyway, so including a preprocessor in the workflow vs. fitting it outside doesn't change 1) the dataset it's being fitted to or 2) the way the final result is presumably evaluated (via the test set).

**Comment by simonpcouch** [comment]
On a re-read of this slide, I think the suggested angle that I was missing is actually "What if we applied the preprocessing pipeline to all of the original data instead of just the training data?" i.e. we _can_ communicate the risk of overfitting using only the knowledge of a train/test split by suggesting that the preprocessing is applied to the whole data frame before splitting (using no knowledge of metrics).

I do wonder if PCA is the clearest example here of "a preprocessing step that includes estimation." Maybe we use centering + scaling instead? That's a separate issue I guess, though.
